algebras/algebra/src/main/scala/endpoints/package.scala

```diff
@@ -0,0 +1,7 @@
+/**
+  *   - '''algebras''' are in the [[algebra]] package,
+  *   - '''interpreters''' are in the other packages (e.g., [[akkahttp]],
```

Comment:
Not sure if this is an issue, but `sbt doc` does not manage to generate links for `[[akkahttp]]`, `[[http4s]]`, and `[[openapi]]`.

Reply:
Thanks for checking that! It works with sbt-unidoc (which is what is used when you run the task `manual/previewSite`)

Reply:
Oh nice! Good to know - I'll check that in the future.